easybuild/easyconfigs/c/cpp-netlib/cpp-netlib-0.13.0-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,48 @@
+easyblock = 'CMakeMake'
+
+name = 'cpp-netlib'
+version = '0.13.0'
+
+homepage = 'https://cpp-netlib.org'
+description = """A collection of open-source libraries for high level network programming."""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+toolchainopts = {'pic': True}
+
+sources = [{
+    'filename': '%(name)s-%(version)s-git.tar.gz',
+    'git_config': {
+        'url': 'https://github.com/cpp-netlib',
+        'repo_name': 'cpp-netlib',
+        'commit': 'cpp-netlib-0.13.0-final',
+        'recursive': True,
+        'keep_git_dir': True,
+    },
+}]
+checksums = ['75ae8c8ca5d2ecc01402b6bd7711d32688c6da5dc865711d14ca6e82aba508ae']
+
+builddependencies = [
+    ('binutils', '2.38'),
+    ('CMake', '3.23.1'),
+    ('Boost', '1.69.0', '-no_mpi'),
+]
+
+dependencies = [
+    # OS dependency should be preferred if the os version is more recent then this version
+    # it's nice to have an up to date openssl for security reasons
+    # ('OpenSSL', '1.1.0e'),
+]
+
+osdependencies = [('openssl-devel', 'libssl-dev', 'libopenssl-devel')]
```

Comment:
@SimonPinches For recent toolchains, we're now using our `OpenSSL` wrapper as a dependency, rather than directly specifying an OS dependency.

So please use this instead:

```python
dependencies = [
    ('OpenSSL', '1.1', '', True),
]
```

Reply:
@boegel, thanks for this information - I've just updated accordingly.